packages/main/src/themes/sap_fiori_3/Avatar-parameters.css

```diff
@@ -0,0 +1,8 @@
+
+:root {
```

Comment:
So only in Quartz Light (sap_fiori_3) the font sizes are different and have the ones below and all the rest themes should have the sizes in the base parameters file?